openml/extensions/sklearn/extension.py

```diff
@@ -476,6 +477,35 @@ def _is_sklearn_flow(cls, flow: OpenMLFlow) -> bool:
             or ',sklearn==' in flow.external_version
         )
 
+    def _get_sklearn_description(self, model: Any, char_lim: int = 1024) -> str:
+        '''Fetches the sklearn function docstring for the flow description
+
+        Parameters
+        ----------
+        model: The sklearn model object
+        char_lim: int, specifying the max length of the returned string
+            OpenML servers have a constraint of 1024 characters for the 'description' field.
+
+        Returns
+        -------
+        string of length <= char_lim
+        '''
+        def match_format(s):
+            return "{}\n{}\n".format(s, len(s) * '-')
+        s = inspect.getdoc(model)
+        if len(s) <= char_lim:
```

Comment:
I'm wondering if we should always cut the parameters and the "read more" as they are not really a description of the algorithm/flow. What do you think?

Reply:
Yeah, I am at this point trying to keep only the descriptions that are there until a "Read More". In case a "Read More" itself is missing in the docstrings, I am then trimming until "Parameters" begin. In effect, I am trying to keep only the meat of the description for the flow description.

Reply:
Hey, I actually meant that you could simple drop the if statement here. Then, the "read more" would automatically be removed even if it the string was shorter than 2014 characters.